src/app/task/task.component.ts

```diff
@@ -0,0 +1,39 @@
+import { TaskService } from '../task.service';
+import { Component, OnInit } from '@angular/core';
+import { Task } from './task';
+import { FormControl, FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'app-task',
+  templateUrl: './task.component.html',
+  styleUrls: ['./task.component.scss']
+})
+export class TaskComponent implements OnInit {
+  taskForm = new FormGroup({
+    name: new FormControl(''),
+    description: new FormControl(''),
+  });
+
+  tasks: Task[] = [];
+
+  constructor(public taskService: TaskService) { }
+
+  ngOnInit() { }
+
```

Comment:
We need to add method getTasks which will retrieve tasks list from TaskService and assign retrieved list to out tasks field in this component. Also we need to invoke this method in onInit and after successfully adding task so we will get updated list of tasks :)